src/Nancy/ModelBinding/DefaultBinder.cs

```diff
@@ -93,20 +92,29 @@ public object Bind(NancyContext context, Type modelType, object instance, Bindin
             var bindingContext =
                 this.CreateBindingContext(context, modelType, instance, configuration, blackList, genericType);
 
-            var bodyDeserializedModel = this.DeserializeRequestBody(bindingContext);
+            var bindingExceptions = new List<PropertyBindingException>();

```

Comment:
Why is this decalration moved up? Are all declarations in the top of the method, or?